Views/MenuWidget.cshtml

```diff
@@ -32,7 +32,7 @@
         </button>
         <div id="@contentIdName" class="collapse navbar-collapse @contentElementName">
             <ul class="navbar-nav me-auto mb-lg-0">
-                @foreach (var menuItem in viewModel.MenuItems)
+                @foreach (var menuItem in viewModel.MenuItems.OrderBy(item => item.Position))

```

Comment:
I'm not a fan of the lexical sorting here. How about something like this?
```csharp
.OrderBy(item => double.TryParse(item.Position, out var number) ? number : double.MaxValue ).ThenBy(item => item.Position)
```

Reply:
Yeah me neither but it looked good enough to both Márk and me, but this would not hurt

Reply:
It didn't look good to me for what I wrote in Teams. If you have more than 10 items (which not impossible for a more complex menu) then it will sort the 10th before the 2nd unless you have the foresight to zero-pad.